modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/cell_renderers/DefaultRenderer.tsx

```diff
@@ -92,6 +92,14 @@ const DefaultRenderer: React.FC<{
 		return <Wrapper options={options}>{value.label}</Wrapper>;
 	}
 
+	if (
```

Comment:
Allow scalar arrays to render as a comma separated list

Reply:
This effectively makes DefaultRenderer an "array-aware" renderer. It's enough for now to cover the cases we want in the Users MVP initiative :tada: